### PR TITLE
dash/datagrid-a11y-quickfix

### PR DESCRIPTION
**Pull request by Vazonik**
Added accessibility quick fix for data grid.

### CODE REVIEW

**Review by sebastianbochan — APPROVED** [review]

File: ts/DataGrid/DataGrid.ts [hunk 1]
```diff
@@ -268,6 +268,8 @@ class DataGrid {
         this.options = merge(DataGrid.defaultOptions, options);
 
         this.gridContainer.style.height = this.getDataGridSize() + 'px';
+        this.gridContainer.role = 'figure';
+        this.gridContainer.tabIndex = 0;
```
Comment (sebastianbochan) [inline ts/DataGrid/DataGrid.ts:272]: We should avoid tabIndex AFAIK, @marvin19 could you confirm?

File: ts/DataGrid/DataGrid.ts [hunk 2]
```diff
@@ -518,6 +520,12 @@ class DataGrid {
         }
 
         this.updateGridElements();
+
+        this.gridContainer.ariaLabel = `Grid with ${
```
Comment (karolkolodziej) [inline ts/DataGrid/DataGrid.ts:524]: Shouldn't it be configurable?

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

File: ts/DataGrid/DataGrid.ts (hunk 2 above)
Comment (sebastianbochan) [reply ts/DataGrid/DataGrid.ts:524]: I think that the whole a11y support should be discussed during the DataGrid "new vision". Right now, it is a temporary solution.

**Review by karolkolodziej — APPROVED** [review]

File: ts/DataGrid/DataGrid.ts (hunk 1 above)
Comment (marvin19) [reply ts/DataGrid/DataGrid.ts:272]: Can be removed

**Review by marvin19 — CHANGES_REQUESTED** [review]

**Review by marvin19 — APPROVED** [review]

**Pull request merged**